fsspec/implementations/tests/test_cached.py

```diff
@@ -1022,7 +1022,7 @@ def test_multi_cache(protocol):
 def test_multi_cat(protocol, ftp_writable):
     host, port, user, pw = ftp_writable
     fs = FTPFileSystem(host, port, user, pw)
-    for fn in {"/file0", "/file1"}:
+    for fn in ("/file0", "/file1"):
```

Comment:
why is a tuple better than a set? It totally doesn't matter in this case, but `contains` should be more efficient for sets, if the set in question is made only once.

Reply:
This `in` isn't a contains though, it's a for loop. Tuple is more efficient to construct and iterate over than a set, I don't feel strongly about this one, happy to disable. It definitely is one of the more controversial rules, happy to disable.

Reply:
It's fine, just wondering for my curiosity.